src/project-3.js

```diff
@@ -1,83 +1,93 @@
 // Do not change any of the function names
 
 const makeCat = (name, age) => {
-  // create a new object with a name property with the value set to the name argument
-  // add an age property to the object with the value set to the age argument
-  // add a method called meow that returns the string 'Meow!'
-  // return the object
+  const cat = {};
+  cat.name = name;
+  cat.age = age;
+  cat.meow = () => 'Meow!';
+  return cat;
 };
 
 const addProperty = (object, property) => {
-  // add the property to the object with a value of null
-  // return the object
-  // note: the property name is NOT 'property'.  The name is the value of the argument called property (a string)
+  object[property] = null;
+  return object;
 };
 
 const invokeMethod = (object, method) => {
-  // method is a string that contains the name of a method on the object
-  // invoke this method
-  // nothing needs to be returned
+  object[method]();
 };
 
 const multiplyMysteryNumberByFive = (mysteryNumberObject) => {
-  // mysteryNumberObject has a property called mysteryNumber
-  // multiply the mysteryNumber property by 5 and return the product
+  return mysteryNumberObject.mysteryNumber * 5;
 };
 
 const deleteProperty = (object, property) => {
-  // remove the property from the object
-  // return the object
+  delete object[property];
+  return object;
 };
 
 const newUser = (name, email, password) => {
-  // create a new object with properties matching the arguments passed in.
-  // return the new object
+  const user = {
+    name,
+    email,
+    password,
+  };
+  return user;
 };
 
 const hasEmail = (user) => {
-  // return true if the user has a value for the property 'email'
-  // otherwise return false
+  if (user.email) {
+    return true;
+  }
+  return false;
 };
 
 const hasProperty = (object, property) => {
-  // return true if the object has the value of the property argument
-  // property is a string
-  // otherwise return false
+  if (object[property]) {
+    return true;
+  }
+  return false;
 };
 
 const verifyPassword = (user, password) => {
-  // check to see if the provided password matches the password property on the user object
-  // return true if they match
-  // otherwise return false
+  if (user.password === password) {
```

Comment:
Here you can actually do this: `return user.password === password;`  The expression gets reduced to `true` or `false` by default